src/components/RouteOption/DockedEbikeRouteOptionDetails.jsx

```diff
@@ -0,0 +1,120 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+function formatTimestamp(isoString) {
+  const date = new Date(isoString);
+  let hours = date.getHours();
+  const minutes = date.getMinutes().toString().padStart(2, "0");
+  const ampm = hours >= 12 ? "pm" : "am";
+  hours = hours % 12 || 12; 
+  return `${hours}:${minutes} ${ampm}`;
+}
+
+function formatDuration(isoDuration) {
+  const regex = /PT(\d+H)?(\d+M)?(\d+S)?/;
+  const matches = isoDuration.match(regex);
+
+  const hours = parseInt(matches[1]) || 0;
+  const minutes = parseInt(matches[2]) || 0;
+  const seconds = parseInt(matches[3]) || 0;
+
+  const totalMinutes = hours * 60 + minutes + Math.round(seconds / 60);
+
+  return `${totalMinutes}m`;
```

Comment:
hmm 🤔 this implementation will represent durations longer than 1 hour as minutes too

good enough for now, I would say (I don't expect total durations for our to be > 1h), but we might need to update it in the future